index.html

```diff
@@ -928,11 +942,23 @@ <h2>
               <var>acceptPromise</var> with that error and terminate this
               algorithm.
               </li>
-              <li>If the user agent has a registered <a>payment handler</a>
-              that supports <var>identifier</var>, then check if the payment
-              handler is authorized and capable of handling the payment request
-              with <var>data</var>. If the check returns in the affirmative,
-              then add payment handler to <var>handlers</var>.
+              <li>Let <var>registeredHandlers</var> be a <a>list</a> of
+              registered <a>payment handlers</a> that are authorized and can
+              handle payment request for <var>identifier</var>.
```

Comment:
What is "can handle [a] payment request"? It seems like another algorithm :-/.

Maybe just "registered payment handlers for the payment method _identifier_" or something?